i18n/locales/source/dashboard/dsls.yml

```diff
@@ -5628,7 +5628,12 @@ en:
       AP_practice_Qs_rqb_118_CSP_4_2_4:
         180 minutes: 180 minutes
         240 minutes: 240 minutes
-        270 minutes: 270 minutes
+        ? '270 minutes
+
+'
```

Comment:
@Hamms I think you may have touched on this in the pass that extra spaces added shouldn't break anything.  Is this true?

Reply:
correct!